pages/common/pio-system.md

```diff
@@ -3,7 +3,7 @@
 > Miscellaneous system commands for PlatformIO.
 > More information: <https://docs.platformio.org/en/latest/core/userguide/system/>.
 
-- Install shell completion for the current shell (supports bash, fish, zsh and powershell):
+- Install shell completion for the current shell (supports Bash, fish, zsh and PowerShell):
```

Comment:
```suggestion
- Install shell completion for the current shell (supports Bash, Fish, ZSH and PowerShell):
```

Reply:
ZSH should be Zsh I think.

> The Z shell (Zsh) is a Unix shell that can be used as an interactive login shell and as a command interpreter for shell scripting.
> \- https://en.wikipedia.org/wiki/Z_shell

> This site provides an index to Zsh information and archives.
> \- https://www.zsh.org/

> Zsh is a shell with lots of features.
> \- https://sourceforge.net/p/zsh/code/ci/master/tree/

```suggestion
- Install shell completion for the current shell (supports Bash, Fish, Zsh and PowerShell):
```

Reply:
Yeah sorry, I saw it as `ZSH` somewhere but I didn't verify.